compiler/luci-interpreter/include/luci_interpreter/core/Tensor.h

```diff
@@ -137,13 +137,36 @@ class Tensor
 
   void resize(const Shape &new_shape);
 
+  void set_data_buffer(uint8_t *buffer)
+  {
+    // It will be implemented when _data changes from unique ptr to raw ptr in next PR
+    (void)buffer;
+  }
+
+  bool is_observable() const { return _is_observable; }
+
+  void make_unobservable() { _is_observable = false; }
```

Comment:
Let's make it more general:
```suggestion
  void set_observable(bool value) { _is_observable = value; }
```
I think this interface is more clear (both setter and getter have "observable" in names)